services/apps/src/apps/clients/metadata.clj

```diff
@@ -0,0 +1,66 @@
+(ns apps.clients.metadata
```

Comment:
Yet more long-term comments! Because it wouldn't be a review from me without at least three. What would you think of us moving away from having a full client in all of these different services, and instead making shared client libraries? I'd expect the shared ones to basically be the "raw" libraries we have for some things, and each service could still have a wrapper which adds any pre/post-processing features they might need. Ideally we'd probably make these be generated clients off of the swagger docs, where we have those.

To put this in the context I'm working from, upcoming data-info migration work for me will involve adding clients to data-info: an apps client for urlupload, a tree-urls client for the manifest, and possibly adding more to the metadata client to move some extra functionality out of terrain too. And of course, all that migration work has constantly added more stuff to terrain's data-info client, which I'd love to mostly just generate instead.